lib/web_ui/lib/src/engine/text/font_collection.dart

```diff
@@ -118,7 +118,7 @@ class _FontManager {
     // family that is not correct CSS syntax. To ensure the font family is
     // accepted on these browsers, wrap it in quotes.
     // See: https://drafts.csswg.org/css-fonts-3/#font-family-prop
-    if (browserEngine == BrowserEngine.webkit || browserEngine == BrowserEngine.firefox) {
+    if (browserEngine == BrowserEngine.firefox) {
```

Comment:
I though we added this originally because it wasn't working in safari with `/` in the name

Reply:
Yeah, we'd need a different explanation from what's in the comment above if we want this change.

Reply:
There's a suggestion on the spec on how to escape the font name properly (regardless of the browser, I guess!)

> To avoid mistakes in escaping, it is recommended to quote font family names that contain white space, digits, or punctuation characters other than hyphens:

It's also important to *not* wrap the [generic font families](https://drafts.csswg.org/css-fonts-3/#generic-font-families), but those would fall under the "things that only contain hyphens" above.

Reply:
The problem is that different browsers interpret the "family name" argument to `FontFace` differently. In Chrome, for example, if you wrap the family name in quotes (i.e., `new FontFace('"Helvetica"', ...)`) it becomes impossible to load that font-face later, because setting the font-family in CSS will remove surrounding quotes. It used to be the case in Safari (and still the case in Firefox, I believe), that they interpret the family-name argument to `FontFace` as something that should be parsed as if it was a CSS String, so they don't accept family names with whitespace or punctuation.

The browsers disagree about how to interpret this argument, see this for reference: https://github.com/w3c/csswg-drafts/issues/3776

Reply:
Thanks all for your comments!

I made a few changes. Now it works for all the browsers (which caused an issue before).

Read the comments for the explanations.

Cheers!